sympy/concrete/products.py

```diff
@@ -163,8 +159,11 @@ class Product(ExprWithIntLimits):
     RisingFactorial(a + 1, -a + b - 1)
     >>> P1 * P2
     RisingFactorial(b, a - b + 1)*RisingFactorial(a + 1, -a + b - 1)
-    >>> simplify(P1 * P2)
-    1
+    >>> simplify((P1 * P2).rewrite(gamma))
```

Comment:
This could be `combsimp(P1 * P2)`.

Reply:
Are you sure that in this case the result should not be a `Piecewise` ?
The variables `a` and `b` have no assumptions, so I am not sure if `1` is the correct result in each case.

Reply:
I think that `a` and `b` should be integers in this context even though there are no assumptions.

Reply:
So should `sin(pi*a)/sin(pi*b)` simplify to `-1**(b - a)` for `a, b` being integers?

Reply:
Using `combsimp`, `P1 * P2` simplifies to `1`, because we have used `piecewise = False` there. I think there is a limitation to the simplification we require here to get the result as `1` automatically.

Reply:
Or rather `gammasimp()` should return a better result when `Euler reflection formula` is being applied on integers.
```
1) For a, b being symbols with no assumptions:

In [110]: y
Out[110]:
Gamma(a)*Gamma(1 - a)
---------------------
Gamma(b)*Gamma(1 - b)

In [111]: y.gammasimp()
Out[111]:
sin(pi*b)
---------
sin(pi*a)

2) For a1, b1 being integers:

In [113]: y1
Out[113]:
Gamma(a1)*Gamma(1 - a1)
-----------------------
Gamma(b1)*Gamma(1 - b1)

In [114]: y1.gammasimp()
Out[114]:
Gamma(a1)*Gamma(1 - a1)
-----------------------
Gamma(b1)*Gamma(1 - b1)

This should be (-1)**(a1 - b1) rather being unevaluated.
```